debs/SPECS/3.11.0/wazuh-agent/debian/rules

```diff
@@ -74,6 +74,9 @@ override_dh_install:
 	rm -rf ${TARGET_DIR}$(INSTALLATION_DIR)/ruleset/sca
 	mkdir -p ${TARGET_DIR}$(INSTALLATION_DIR)/ruleset/sca
 
+	# Copying ossec-init.conf
+	${SCRIPTS_DIR}/gen_ossec.sh init agent ${DIR} > ${DIR}/etc/ossec-init.conf
```

Comment:
This file has already been created by the `install.sh` while it was executed. In v3.10.0 rules files you are copying the file but here you are only creating it but it isn't copied.

Reply:
Error of previous tests